### PR TITLE
Method in server to return Cache statistics

### PR DESCRIPTION
**Pull request by sakshia125**
Created a new method in server which returns Cache statistics (Cache Hit Rate and Miss Rate).

### CODE REVIEW

File: rubix-bookkeeper/src/main/thrift/bookkeeper.thrift [hunk 1]
```diff
@@ -1,10 +1,14 @@
 namespace java com.qubole.rubix.bookkeeper
+namespace python com.qubole.rubix.bookkeeper
 
 typedef i64 long
+typedef i32 int
```
Comment (stagraqubole) [inline rubix-bookkeeper/src/main/thrift/bookkeeper.thrift]: is this needed?

Comment (sakshia125) [inline rubix-bookkeeper/src/main/thrift/bookkeeper.thrift]: no, i forgot to delete it.

File: rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java [hunk 2]
```diff
@@ -93,6 +104,15 @@ public void setAllCached(String remotePath, long fileLength, long startBlock, lo
         }
     }
 
+    @Override
+    public Map getCacheStats()
+    {
+        Map<String, Double> stats = new HashMap<String, Double>();
+        stats.put("Cache Hit Rate", (double) (cachedRequests/(totalRequests)));
```
Comment (stagraqubole) [inline rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java]: You are loosing precision here. Move the `double` cast inside, ear cachedRequests

File: rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java [hunk 3]
```diff
@@ -93,6 +104,15 @@ public void setAllCached(String remotePath, long fileLength, long startBlock, lo
         }
     }
 
+    @Override
+    public Map getCacheStats()
+    {
+        Map<String, Double> stats = new HashMap<String, Double>();
+        stats.put("Cache Hit Rate",  ((double)cachedRequests/totalRequests));
```
Comment (stagraqubole) [inline rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java]: leave space after (double) ... we need to add checkstyle too, will open issue for that

**Pull request merged**